modules/pel/peltool/peltool.py

```diff
@@ -713,68 +713,96 @@ def main():
-    parser.add_argument('-D', '--delete-all', dest='deleteAll',
-                        action='store_true', help='Delete all PELs')
-    parser.add_argument('-S', '--severities', nargs='+', choices=list(severityGroupValues.keys()), 
-                        help=f'Filter by severity value.')
-    parser.add_argument('-x', '--hex',
-                        action='store_true', help='Display PEL(s) in hexdump instead of JSON')
-
-    if not inBMC:
-        parser.add_argument('-p', '--path',
-                        dest='path', help='Specify path to PELs')
-    else:
-        # peltool -A/--archive is specific to BMC only.
-        parser.add_argument('-A', '--archive',
-                        action='store_true', help='List or display archived PELs')
+    parser.add_argument('-d', '--delete', dest='IDToDelete',
+                        metavar='<EID>', help='Delete a PEL based on its EID') 
+    parser.add_argument('-D', '--delete-all', dest='deleteAll', action='store_true',
+                        help='Delete all PELs')
+    parser.add_argument('-i', '--id', dest='pelID', metavar='<EID>',
+                        help='Display a PEL based on its EID')
+    parser.add_argument('--bmc-id', dest='bmcID',
+                        metavar='<BMC_Event_Log_Id>',
+                        help='Display a PEL based on its BMC Event Log ID')
+    parser.add_argument('--plid', dest='plID', metavar='<Platform_Log_Id>',
+                        help='Display PELs based on its Platform Log ID')
+    parser.add_argument('--src', dest='src', metavar='<System_Refrence_Code>',
+                        help='Display PELs based on its System Reference Code')
+    parser.add_argument('--src-exclude', dest='src_exclude_file',
+                        metavar='<SRC_Exclude_file>', help='Display PELs excluding SRCs from the file')
+    # Exclusive peltool options
+    pelExclusive = parser.add_argument_group('Exclusive options to get PELs',
+                                             'Use below options along with -l/--list, '
+                                             '-a/--all, -n/--show-pel-count options.'
+                                             'Check example usage')
+    pelExclusive.add_argument('-s', '--serviceable', action='store_true',
+                              help='Get Serviceable PELs')
+    pelExclusive.add_argument('-N', '--non-serviceable', action='store_true',
+                              help='Get Non-Serviceable PELs')
+    pelExclusive.add_argument('-H', '--hidden', action='store_true',
+                              help='Get count of hidden PELs only')
+    pelExclusive.add_argument('-t', '--termination', dest='critSysTerm', action='store_true',
+                              help='List only critical system terminating PELs')
+    pelExclusive.add_argument('-S', '--severities', nargs='+',
+                              choices=list(severityGroupValues.keys()),
```

Comment:
I don't see the actual values in the output anymore:
```
  -S severity1 severity2 ... [severity1 severity2 ... ...], --severities severity1 severity2 ... [severity1 severity2 ... ...]
                        Filter by severity value
```
It'd be good to get it to show that choices list again so users know what they can put here.